.github/workflows/obs-build-smoke-test.yml

```
@@ -0,0 +1,40 @@
+name: build in-obs smoke test
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+  workflow_dispatch:
+
+jobs:
+  build-packit-in-obs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+      - name: Create osc config
+        env:
+          OBS_USER: ${{ secrets.OBS_USERNAME }}
+          OBS_PASSWORD: ${{ secrets.OBS_PASSWORD }}
+
+        run: |
+          mkdir -p $HOME/.config/osc/
+          cat << EOF > $HOME/.config/osc/oscrc
+          [general]
+          apiurl=https://api.opensuse.org
+          [https://api.opensuse.org]
+          user=$OBS_USER
+          pass=$OBS_PASSWORD
```

```suggestion
          OBS_USERNAME: ${{ secrets.OBS_USERNAME }}
          OBS_PASSWORD: ${{ secrets.OBS_PASSWORD }}

        run: |
          mkdir -p $HOME/.config/osc/
          cat << EOF > $HOME/.config/osc/oscrc
          [general]
          apiurl=https://api.opensuse.org
          [https://api.opensuse.org]
          user=$OBS_USERNAME
          pass=$OBS_PASSWORD
```

Would be nice for consistency..

---

sure. would fix that right away